src/calibre/ebooks/metadata/sources/ozon.py

```diff
@@ -71,7 +71,10 @@ def get_book_url(self, identifiers):  # {{{
     # }}}
 
     def create_query(self, log, title=None, authors=None, identifiers={}):  # {{{
-        from urllib import quote_plus
+        try:
+            from urllib import quote_plus
+        except ImportError:
+            from urllib.parse import quote_plus
```

Comment:
Can you please first try the python3 version, then fall back on the python2 version?